app/javascript/src/Policies/components/SchemaEditor.jsx

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+import * as React from 'react'
+import { useState } from 'react'
+import { UnControlled as CodeMirror } from 'react-codemirror2'
+import { fromJsonString, toJsonString } from 'Policies/util'
+import 'codemirror/mode/javascript/javascript'
+import ApicastManifest from 'Policies/apicast-manifest'
+import Ajv from 'ajv'
+import type { Schema } from 'Policies/types/Policies'
+
+const ajv = new Ajv().addSchema(ApicastManifest, 'ApicastManifest')
+
+const CM_OPTIONS = {
```

Comment:
Maybe add types for this object and it's keys

Reply:
This object is for the settings of codemirror, it varies widely. The only way would be adding the `Object` type or assign somethiing like `CodeMirrorSettings = Object`. Still not sure if it would be useful since it's used only there. Wdyt?

Reply:
Second thought, if they don't provide a type for its options, we shouldn't.

Reply:
OK